src/kOS/Suffixed/VesselTarget.cs

```diff
@@ -464,6 +465,13 @@ private void InitializeSuffixes()
             AddSuffix("MESSAGES", new NoArgsSuffix<MessageQueueStructure>(() => GetMessages()));
         }
 
+        public global::Part GetControlPart()
+        {
+            global::Part res = Vessel.GetReferenceTransformPart(); //this can actually be null if "control from here" has never been used
```

Comment:
Incidentally, that was the first craft I used this on after uploading initial the version here. The tests I did before showed no issues. Testing luck.
It seems a bit strange that the function just returns null instead of root, but maybe it's used somewhere as "if there's something better than root, switch control to that, otherwise control from root"